showcase/tests/integration/components/hds/layout/grid/index-test.js

```diff
@@ -57,6 +59,27 @@ module('Integration | Component | hds/layout/grid/index', function (hooks) {
       .hasStyle({ '--hds-layout-grid-column-min-width': '200px' });
   });
 
+  // COLUMN WIDTH
```

Comment:
[suggestion] I see four tests for three logical conditions; maybe we could unify them, and match the "logic" in the JS so it's easier to follow the test?